server/init.go

```diff
@@ -207,6 +200,68 @@ func InitCmd(ctx *Context, cdc *wire.Codec, appInit AppInit) *cobra.Command {
 	return cmd
 }
 
+func initWithConfig(ctx *Context, cdc *wire.Codec, appInit AppInit, config *cfg.Config, gaiaInitConfig InitConfig) (
```

Comment:
nothing should use the naming "gaia" outside of the gaia example directory - should rename `gaiaInitConfig` to something more generic that's fitting for it's usage

Reply:
initConfig it is.